hbase-server/src/main/java/org/apache/hadoop/hbase/master/CatalogJanitor.java

```diff
@@ -672,6 +672,11 @@ private void checkServer(byte [] metaTableRow, RegionLocations locations) {
         if (sn == null) {
           continue;
         }
+        // skip the offline regions which belong to disabled table.
+        if (services.getTableStateManager()
+            .isTableState(location.getRegion().getTable(), TableState.State.DISABLED)) {
```

Comment:
Oh. I see. I didn't see it in my reproduction of the issue.

Could be DISABLED or DISABLING. Thats what we check in other places IIRC.

Reply:
There already a method which call isTableDisabled. Let me change to use that one.